pantheon/src/main/java/tech/pegasys/pantheon/util/BlockImporter.java

```diff
@@ -162,7 +162,7 @@ private void extractSignatures(final Block block) {
     final BlockHeaderValidator<C> blockHeaderValidator = protocolSpec.getBlockHeaderValidator();
     final boolean validHeader =
         blockHeaderValidator.validateHeader(
-            header, previousHeader, context, HeaderValidationMode.FULL);
+            header, previousHeader, context, HeaderValidationMode.DETACHED_ONLY);
```

Comment:
why did the validation mode need to be changed?

Reply:
This was a bug in Pantheon which was hidden up until this change - basically they do async block validation (thus block 2 may be validated before block 1) - this is illegal in IBFT, as we must validate a block fully with respect to its parent.

Reply:
Checked and this was changed to fix a bug "the validation rule was a hidden bug in Pantheon - the BlockImporter validates blocks ASYNC - meaning, out of order :disappointed: Which IBFT can no longer do!
So when doing ASYNC, you MUST do only detached rules as the initial validation, then do the rest at the end"